training_config/mnli_roberta.jsonnet

```diff
@@ -0,0 +1,66 @@
+local transformer_model = "roberta-large";
+local transformer_dim = 1024;
+local cls_is_last_token = false;
+
+{
+  "dataset_reader":{
+    "type": "snli",
+    "tokenizer": {
+      "type": "pretrained_transformer",
+      "model_name": transformer_model
+    },
+    "token_indexers": {
+      "tokens": {
+        "type": "pretrained_transformer",
+        "model_name": transformer_model,
+        "max_length": 512
+      }
+    }
+  },
+  "train_data_path": "https://allennlp.s3.amazonaws.com/datasets/multinli/multinli_1.0_train.jsonl",
+  "validation_data_path": "https://allennlp.s3.amazonaws.com/datasets/multinli/multinli_1.0_dev_matched.jsonl",
+  "test_data_path": "https://allennlp.s3.amazonaws.com/datasets/multinli/multinli_1.0_dev_mismatched.jsonl",
+  "model": {
+    "type": "basic_classifier",
+    "text_field_embedder": {
+      "token_embedders": {
+        "tokens": {
+          "type": "pretrained_transformer",
+          "model_name": transformer_model,
+          "max_length": 512
+        }
+      }
+    },
+    "seq2vec_encoder": {
+       "type": "cls_pooler",
+       "embedding_dim": transformer_dim,
+       "cls_is_last_token": cls_is_last_token
+    },
+    "feedforward": {
```

Comment:
I was just checking Anthony's results versus the numbers you're reporting.  You're about 1.5 points below Anthony's MNLI result, and one point below the published number.  I looked at Anthony's code, and it looks like the only difference is that he doesn't have this feedforward layer in there.  What happens if you remove this?

Reply:
RoBERTa actually has this feedforward layer (using the same model as BERT) https://github.com/huggingface/transformers/blob/908fa43b543cf52a3238129624f502240725a6a6/src/transformers/modeling_bert.py#L426-L438

Reply:
And I tried w/ vs. w/o this on SST -- didn't make a difference.